src/modules/node-clean.ts

```diff
@@ -25,7 +26,16 @@ const NodeClean = {
             }' directory in ${shell.pwd()}...`
         );
 
-        shell.rm("-rf", Constants.NODE_MODULES);
```

Comment:
Not 100% sure if `rm -rf` is cross platform - there might be a reason `shell.rm` is provided.  https://github.com/shelljs/shelljs/blob/master/src/rm.js